CMakeLists.txt

```diff
@@ -179,17 +179,24 @@ if(SIMENG_ENABLE_TESTS)
   if (NOT "AArch64" IN_LIST LLVM_TARGETS_TO_BUILD)
     message(FATAL_ERROR "LLVM was built without AArch64 target")
   endif()
+  if (NOT "RISCV" IN_LIST LLVM_TARGETS_TO_BUILD)
+      message(FATAL_ERROR "LLVM was built without RISCV target")
+  endif()
 
   ## Setup googletest ##
   FetchContent_MakeAvailable_Args(googletest EXCLUDE_FROM_ALL)
   enable_testing()
 
   add_subdirectory(test)
   # saves us from having to build all targets before running the tests
-  add_custom_target(test-all
-    COMMAND ${CMAKE_CTEST_COMMAND}
-    DEPENDS unittests regression-aarch64
-  )
+#  add_custom_target(test-all
+#    COMMAND ${CMAKE_CTEST_COMMAND}
+#    DEPENDS unittests regression-aarch64
+#  )
+    add_custom_target(test-all
+        COMMAND ${CMAKE_CTEST_COMMAND}
+        DEPENDS unittests regression-riscv
+    )
```

Comment:
Add the aarch64 tests back in before merging to main? Presume we can have both aarch64 and RISC-V

Reply:
Yeah, could just have `DEPENDS unittests regression-aarch64 regression-riscv` right?

Reply:
Both sets of tests run when using ```--target test``` as is. Have incorporated Mutalib's suggestion